plugins/mayastor.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: mayastor
+spec:
+  version: v2.5.0
+  homepage: https://mayastor.gitbook.io/
+  shortDescription: Provides commands for OpenEBS Mayastor.
+  description: |
+    This plugin allows you to manage Mayastor volumes and upgrades.
+  platforms:
+  - selector:
+      matchLabels:
+        os: darwin
+        arch: amd64
```

Comment:
You should look into adding darwin/arm64 as that's the only macbooks sold nowadays practically nowadays.